core/common/src/main/java/alluxio/AbstractThriftClient.java

```diff
@@ -165,7 +166,8 @@
       }
     } while (retryPolicy.attemptRetry());
 
-    LOG.error("Failed after " + retryPolicy.getRetryCount() + " retries.");
+    //LOG.error("Failed after " + retryPolicy.getRetryCount() + " retries.");
+    LOG.error("Failed after {} retries.", retryPolicy.getRetryCount());
```

Comment:
I'm sorry about it. I have deleted the modified code now. Please check this. Thanks.